proto/xudrpc.proto

```diff
@@ -467,6 +467,15 @@ message Order {
   uint64 hold = 10 [json_name = "hold"];
 }
 
+message Currency {
```

Comment:
I'm also thinking it would be nice to have the swap client used by the currency here, either lnd or raiden for now. We include this when we add the currency, so it makes sense to have it when we list them.